master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Application layer policy tutorial
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/tutorials/app-layer-policy/'
+---
+
+The included demo sets up a microservices application, then demonstrates the use of application
+layer policy to mitigate some common threats.
+
+## Pre-requisites
```

Comment:
Generally, we do not use a hyphen here in this word.

Reply:
I will also fix in the star policy tutorial